source/game/system/RaceConfig.hh

```diff
@@ -32,13 +32,22 @@ public:
         enum class GameMode {
             Time_Trial = 2,
             Ghost_Race = 5,
+            LocalGameModeMax = 6,
         };
 
         void init();
 
+        inline bool isOnlineGameMode() const {
```

Comment:
This should be removed, and any logic which depends on it should be removed and assumed to always be offline.